### PR TITLE
Preview new gpu reqs

### PR DESCRIPTION
**Pull request by ChristinaLK**
builds on #425; if this is merged, that can be closed. 

Waiting on full pool upgrade AND consultation with facilitators about syntax and user transition before merging.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
https://chtc.github.io/web-preview/preview-new-gpu-reqs/uw-research-computing/gpu-jobs

**Comment by lmichael107** [comment]
@CannonLock can you look into the failed links?

File: uw-research-computing/gpu-jobs.md [hunk 1]
```diff
@@ -129,12 +129,15 @@ like to submit by using the submit file option below.
...
 	"short" jobs because you will be able to have more jobs running at once. 
 
-- **Request Specific GPUs or CUDA Functionality (optional)**: If your software or code requires a specific version of CUDA, a certain
-type of GPU, or has some other special requirement, you will need to add
-a "requirements" statement to your submit file that uses one of the
-attributes shown above. If you want a certain class of GPU, use `CUDACapability`:
+- **Request Specific GPUs or CUDA Functionality Using `require_gpus` (optional)**: If your software or code requires a certain
+type of GPU, or has some other special requirement, there is a special submit file line 
+to request these capabilities, `require_gpus`. Each GPU attribute that can be requested 
+in this way is structured as `GPUs_feature`. You can request specific features by using
```
Comment (lmichael107) [inline uw-research-computing/gpu-jobs.md:135]:
```suggestion
in this way is structured as `GPUs_<Feature>`. You can request specific features by using 
```

File: uw-research-computing/gpu-jobs.md [hunk 2]
```diff
@@ -129,12 +129,15 @@ like to submit by using the submit file option below.
...
-type of GPU, or has some other special requirement, you will need to add
-a "requirements" statement to your submit file that uses one of the
-attributes shown above. If you want a certain class of GPU, use `CUDACapability`:
+- **Request Specific GPUs or CUDA Functionality Using `require_gpus` (optional)**: If your software or code requires a certain
+type of GPU, or has some other special requirement, there is a special submit file line 
+to request these capabilities, `require_gpus`. Each GPU attribute that can be requested 
+in this way is structured as `GPUs_feature`. You can request specific features by using 
+the feature name in the `require_gpus` statment.  For example, if you want a certain 
+class of GPU, represented by 
+the attribute `GPUs_Capability`, your `require_gpus` statement would look like this:
```
Comment (lmichael107) [inline uw-research-computing/gpu-jobs.md:138]: Do we (and can we help the user) understand why you use 'GPUs_Capabilit' in the preceding text, but 'Capability' in the example?

**Review by lmichael107 — CHANGES_REQUESTED** [review]

File: uw-research-computing/gpu-jobs.md (hunk 2 above)
Comment (ChristinaLK) [reply uw-research-computing/gpu-jobs.md:138]: That's how TJ coded it -- that the require piece just needs the feature and not the GPU prefix. i can try to make that more clear.

**Comment by CannonLock** [comment]
@lmichael107 Just made a PR to add skip these links as the are all false negatives.

**Pull request merged**